salt/utils/schedule.py

```diff
@@ -1203,15 +1202,15 @@ def _handle_range(job, data):
                             data['_error'] = ('Invalid date string for start. '
                                               'Ignoring job {0}.'.format(job))
                             log.error(data['_error'])
-                            return data
+                            return None
```

Comment:
Just `return` no need to `return None`. Linters will likely complain.

Reply:
Or just remove the returns.

Reply:
Returns are needed to exit early, I'll clean up the nones